src/Utilities/MPIExceptionWrapper.hpp

```diff
@@ -0,0 +1,55 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2020 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef MPIEXECPTIONWRAPPER_H
+#define MPIEXECPTIONWRAPPER_H
+
+#include "Message/Communicate.h"
+
+namespace qmcplusplus
+{
+
+class MPIExceptionWrapper
+{
+public:
+  MPIExceptionWrapper() {};
```

Comment:
In this case, I think there is no need of explicit empty constructor.

Reply:
Please also delete the std::cout in `~MCPopulation` and probably just delete the whole empty destructor.